frontend/src/app/modules/work_packages/routing/wp-list-view/wp-list-view.component.ts

```diff
@@ -105,6 +113,15 @@ export class WorkPackageListViewComponent extends UntilDestroyedMixin implements
       this.noResults = query.results.total === 0;
       this.cdRef.detectChanges();
     });
+
+    // Scroll into view the card/row that represents the last selected WorkPackage
+    // so when the user clicks a WP from a wp-split-view-entry (list of wp) and then
+    // clicks on the 'back button', the last selected card is visible on this list.
+    const selectedElement = this.elementRef.nativeElement.querySelector('.-checked');
```

Comment:
It looks pretty unsafe to simply check the for the modifier class `.-checked` only. I'd also check for the element's class as well. We try to apply BEM http://getbem.com/naming/. So the modifier class alone is not specific enough.